PosProject/src/main/java/com/pos/posproject/common/ProductCatalogDetails.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.pos.posproject.common;
+
+/**
+ *
+ * @author Monica
+ */
+public class ProductCatalogDetails implements java.io.Serializable {
+     private Integer id;
+    private String productCatalogName;
+
```

Comment:
We should also add a collection of product details here, because on the frontend we want to be able to have access to the products from the catalogue